cumulusci.yml

```diff
@@ -99,21 +99,21 @@ tasks:
 
     robot:
         options:
-            suites: robot/HEDAP/tests
+            suites: robot/EDAP/tests
```

Comment:
@dashattuck  Should we drop the "P" in the folder names as well?

Reply:
yes - I just need to modify the library paths in all the files - but that's easy to do

Reply:
We have paths like the following, in every script:

Resource        robot/EDAP/resources/EDA.robot

Reply:
should I just leave those as-is?  As in, put them all back to:  
Resource        robot/HEDAP/resources/HEDA.robot

Reply:
Same comment as below: Rename the director to `robot/EDA` and revise any library references to match.

Reply:
Directory has been renamed to be EDA.  All library (resource) references now contain EDA as the folder name, instead of EDAP